fledging.qmd

```diff
@@ -6,7 +6,81 @@ Fledging is a goal of these JupyterHubs, so that users who have had success
 and want to continue their adventures in cloud computing, have the knowledge
 and resources to develop their own solutions.
```

Comment:
I think we might want a little intro contextualizing how we see and experience the challenge of fledging? As discussed in the slack, this is somewhat the inverse of how 'cloud' is often presented (where new users begin in laptop-based installations first and must then learn how (and when) to migrate those workflows to cloud.  The idea of users who can run analyses on the cloud while actually not knowing how or even if they could run it on their own laptop can frankly be mind-blowing to a certain slice of even quite sophisticated computational users.